assets.sy

```diff
@@ -1,19 +1,23 @@
-arena := ("", 0)
-font := ("", 0)
-player1_back := ("", 0)
-player1_front := ("", 0)
-player1_side := ("", 0)
-player2_back := ("", 0)
-player2_front := ("", 0)
-player2_side := ("", 0)
-puck := ("", 0)
+empty_asset :: ("", 0)
+
+arena := empty_asset
+arrow := empty_asset
+font := empty_asset
+player1_back := empty_asset
+player1_front := empty_asset
+player1_side := empty_asset
+player2_back := empty_asset
+player2_front := empty_asset
+player2_side := empty_asset
+puck := empty_asset
 
 player1 : [(str, int)] = []
 
 player2 : [(str, int)] = []
 
 load_assets :: fn {
     arena = l_load_image' "res/rink.png", (200, 100)
+    arrow = l_load_image' "res/arrow.png", (5, 7)
```

Comment:
The scaling off the arrow is off. This needs to be fixed where it's drawn, since the sprite isn't in a 1x1 aspect ratio.